cmd/cluster-agent/api/v1/languagedetection/telemetry.go

```diff
@@ -0,0 +1,36 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2023-present Datadog, Inc.
+
+//go:build kubeapiserver
+
+package languagedetection
+
+import "github.com/DataDog/datadog-agent/pkg/telemetry"
+
+const subsystem = "language_detection_dca_handler"
+
+var (
+	commonOpts = telemetry.Options{NoDoubleUnderscoreSep: true}
+)
+
+var (
+	// OkResponses tracks the number the request was processed successfully
+	OkResponses = telemetry.NewCounterWithOpts(
+		subsystem,
+		"ok_response",
+		[]string{},
+		"Tracks the number of times request processing succeeds",
+		commonOpts,
+	)
+
+	// ErrorResponses tracks the number of times request processsing fails
+	ErrorResponses = telemetry.NewCounterWithOpts(
+		subsystem,
+		"fail_response",
+		[]string{},
+		"Tracks the number of times request processing fails",
+		commonOpts,
+	)
```

Comment:
Is there a way we could avoid creating Global variables for the `languagedetection` package? Are this variables used outside of the `languagedetection` package? If not we might be able to instantiate them when the handler `InstallLanguageDetectionEndpoints` gets called.

Reply:
Hey @GustavoCaso 

Sorry didn't pay attention to your comment before merging.

Will take your suggestion into consideration in a separate PR

Thanks 🙇‍♂️